qa/rpc-tests/cfund-proposal-reorg.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Navcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import NavCoinTestFramework
+from test_framework.cfund_util import *
+
+import urllib.parse
+
+class CommunityFundProposalReorg(NavCoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+    def setup_network(self, split=False):
+        self.nodes = []
+
+        self.nodes.append(start_node(0, self.options.tmpdir, []))
+        self.nodes.append(start_node(1, self.options.tmpdir, []))
+
+        connect_nodes_bi(self.nodes, 0, 1)
+
+        self.is_network_split = split
+
+    def run_test(self):
+
+        activate_cfund(self.nodes[0])
+        sync_blocks(self.nodes)
+
+        rawproposal = self.nodes[0].createproposal(self.nodes[0].getnewaddress(), 10, 36000, "test", 50, True)
+
+        # disconnect the nodes and generate the payout on each node
```

Comment:
does this generate the payment request on each node, or just the proposal? Is it worth adding a test to cover the payment request as well?